wordpress-mobile/WordPress-Android
# Fix crash when contacting customer support within the editor

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/6130.
Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/18804.

This PR addresses a crash related to contacting customer support from the Gutenberg editor. The custom support dialog needs to be called from the main thread. Since the calls from the editor happen on a different thread, we have to ensure that the call to display the customer support dialog is done in the main thread.

**To test:**

1. Open a post/page.
2. Tap on the three dots button located at the top bar.
3. Tap on "Help & Support"
4. Tap on the "Contact support" button.
5. Observe the customer support dialog is displayed and no crash is produced.

## Regression Notes
1. Potential unintended areas of impact
N/A

3. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

4. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
